Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -377,6 +377,13 @@ extension HTTPClient {
         public var headers: HTTPHeaders
         /// Response body.
         public var body: ByteBuffer?
+        /// The history of all requests and responses in redirect order.
+        public var history: [RequestResponse]
+
+        /// The target URL (after redirects) of the response.
+        public var url: URL? {
+            self.history.last?.request.url
```

Comment:
Do we think `nil` is actually possible here?

Reply:
as mentioned above, but copying here: Not from any `HTTPClient.Response` vended by the library, but since there are already public initializers, it will be `nil` for any library users that have been creating their own `HTTPClient.Response`.